test/pkg/test/framework.go

```diff
@@ -313,6 +318,36 @@ func (env *Framework) GetTargets(ctx context.Context, targetGroup *vpclattice.Ta
 	return found
 }
 
+func (env *Framework) ExpectEventuallyVpcOfCurrentClusterAssociateWithServiceNetwork(ctx context.Context, sn *vpclattice.ServiceNetworkSummary) {
+	currentClusterVpcId := os.Getenv("CLUSTER_VPC_ID")
+	fmt.Println("ExpectEventuallyVpcOfCurrentClusterAssociateWithServiceNetwork currentClusterVpcId:", currentClusterVpcId)
```

Comment:
1. please use log not fmt 
2. ExpectEventuallyVpcOfCurrentClusterAssociateWithServiceNetwork can you use shorter names please?
 expect - eventually - these words are implementation details, not a function name

Reply:
ok, will change to log

I think using `expect - eventually` could make the caller more clearer to know that, within this function, it use Expect() to do some assertion and use eventually to do the retry

Reply:
In this case I would use name like EnsureSnVpcAssociation. Function returns something, for example error or bool if there is no association, then you propagate this to caller into a main test function, and to eventually block there.